src/metabase/lib/metadata/jvm.clj

```diff
@@ -340,16 +340,19 @@
   (t2/select :metadata/table
              :db_id           database-id
              :active          true
-             :visibility_type [:not-in #{"hidden" "technical" "cruft"}]))
+             {:where [:or
+                      [:= :visibility_type nil]
+                      [:not-in :visibility_type #{"hidden" "technical" "cruft"}]]}))
 
 (defn- metadatas-for-table [metadata-type table-id]
   (case metadata-type
     :metadata/column
     (t2/select :metadata/column
                :table_id        table-id
                :active          true
-               :visibility_type [:not-in #{"sensitive" "retired"}])
-
+               {:where [:or
+                        [:= :field/visibility_type nil]
+                        [:not-in :field/visibility_type #{"sensitive" "retired"}]]})
```

Comment:
These look bugfixes, do we have regression tests?

Reply:
These are fixes for why `lib.metadata/tables` didn't return anything on JVM (`:not-in` fails for NULL columns). I don't think we actually do that anywhere outside of REPL sessions, though?

Reply:
[This](https://github.com/metabase/metabase/blob/master/test/metabase/lib/test_util.cljc#L243) is the only call anywhere, it seems.